pkg/kapp/config/resource_matchers.go

```diff
@@ -56,7 +56,7 @@ type KindNamespaceNameMatcher struct {
 }
 
 type HasAnnotationMatcher struct {
-	Keys []string
+	KeyValues []ctlres.KeyValue
```

Comment:
this is a backwards incompatible change. let's add KeyValues as an additional key instead of replacing Keys.

Reply:
could you also separate keyvalues change into its own PR?

Reply:
Yep - I'll get this split out in to a non breaking change PR of its own. Just that both of these changes were required for use to try start using `kapp` on the jenkins-x project hence my laziness ;)